framework/files/usr/lib/systemd/system/elemental-register.timer

```diff
@@ -1,6 +1,9 @@
 [Unit]
 Description=Run elemental-register every 30 minutes
 Documentation=https://elemental.docs.rancher.com
+# Do not run the timer for installation media or recovery system
+ConditionPathExists=!/run/elemental/live_mode
+ConditionPathExists=!/run/elemental/recovery_mode
```

Comment:
These conditions match with the conditions that were added to the manual start within the deleted file.